src/main/java/pl/nogacz/chess/application/menu/Undo.java

```diff
@@ -0,0 +1,86 @@
+package pl.nogacz.chess.application.menu;
+
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+import jdk.internal.net.http.common.SubscriberWrapper;
+import pl.nogacz.chess.board.Board;
+import pl.nogacz.chess.board.Coordinates;
+import pl.nogacz.chess.application.ChessNotation;
+import pl.nogacz.chess.pawns.PawnClass;
+
+public class Undo{
+
+    public Undo(){
+        Coordinates oldCoordinate;
+        Coordinates newCoordinate;
+        boolean playerKicked=false;
+        boolean computerKicked=false;
+
+        Alert alert = new Alert(Alert.AlertType.NONE);
+        alert.setTitle("JavaChess");
+        alert.setContentText("Undo last move");
+
+        String lastMove=ChessNotation.removeMovement();
+
+        String computer=lastMove.substring(lastMove.indexOf(" ")+1);
+        if((computer.charAt(0)=='H') || (computer.charAt(0)=='S') || (computer.charAt(0)=='K') || (computer.charAt(0)=='W') || (computer.charAt(0)=='G'))
+            computer=computer.substring(1);
+        if(computer.charAt(0)=='x'){
+            playerKicked=true;
+            computer=computer.substring(1);
+        }
+        String computerOld=computer.substring(0,2);
```

Comment:
Tüm kodda kodun geneli ile daha consistent bir spacing kullan.

Örneğin,

```computerOld=computer.``` yerine

```computerOld = computer.``` olmalı.

Bu dediğimi dün değişiklik yaptığın satırlara uygula parçalar arası bir boşluk)